src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -480,6 +480,25 @@ public virtual ScriptBlock InitializationScript
 
         private ScriptBlock _initScript;
 
+
+        /// <summary>
+        /// Working directory of the process.
```

Comment:
```suggestion
        /// Gets or sets a working directory of the process.
```

Reply:
Thanks for the review @iSazonov
fixed in https://github.com/davinci26/PowerShell/commit/396369a7fe660166cd829e4a99f63fd913760e9c